index.html

```diff
@@ -46,6 +46,204 @@ <h1>I'm Naan.
             </div>
         </headline>
         <!-- END OF HEADLINE SECTION-->
+
+        <!--Works Section-->
+        <!--Work 1-->
+        <div class="primary_text">
```

Comment:
As you titled the Comment as `Works Section`, this is a section and should be encapsulated on a `<section>` tag. So kindly change 
`<div class="primary_text">` to
`<section class="primary_text">`